assets/js/components/Vehicle.vue

```diff
@@ -248,7 +248,7 @@ export default {
 		},
 		fmtEnergy(value) {
 			const inKw = value == 0 || value >= 1000;
-			return this.fmtKWh(value, inKw);
+			return this.fmtWh(value, inKw ? WATT_FORMAT.KW : WATT_FORMAT.AUTO);
```

Comment:
```suggestion
			return this.fmtWh(value, value == 0 ? WATT_FORMAT.KW : WATT_FORMAT.AUTO);
```

The `value >= 1000` can be removed as its already handled in `AUTO`.